tRecorderApi/api/models/project.py

```diff
@@ -1,458 +1,453 @@
-    #                 .filter(chunk__endv=chunk.endv) \
-    #                 .filter(chunk__chapter__project__is_source=True) \
-    #                 .first()
-    #             if source_take:
-    #                 if source_take.markers:
-    #                     source_take.markers = json.loads(source_take.markers)
-    #                 else:
-    #                     source_take.markers = {}
-
-    #                 source_dic["take"] = model_to_dict(source_take, fields=[
-    #                     "markers","location","id"
-    #                 ])
-    #                 source_dic["take"]["version"] = source_take.chunk.chapter.project.version
-    #                 chunk_dic["source"] = source_dic"""
-
-    #         # Include takes
-    #         takes_list = []
-    #         for take in chunk.take_set.all():
-    #             if "is_publish" in data:
-    #                 if take.is_publish != data["is_publish"]:
-    #                     continue
-
-    #             take_dic = {}
-
-    #             # Include author of file
-    #             try:
-    #                 take_dic["user"] = model_to_dict(
-    #                     take.user, fields=["name", "agreed", "picture"])
-    #             except:
-    #                 pass
-
-    #             # Include comments for chunks
-    #             take_dic["comments"] = []
-    #             # for cmt in Comment.objects.filter(content_type=take.id).values():
-    #             for cmt in take.comments.all():
-    #                 comm_dic = {}
-    #                 comm_dic["comment"] = model_to_dict(
-    #                     cmt, fields=["id", "location", "date_modified"])
-    #                 # Include author of comment
-    #                 try:
-    #                     comm_dic["user"] = model_to_dict(
-    #                         cmt.user, fields=["name", "agreed", "picture"])
-    #                 except:
-    #                     pass
-    #                 take_dic["comments"].append(comm_dic)
-
-    #             # Parse markers
-    #             if take.markers:
-    #                 take.markers = json.loads(take.markers)
-    #             else:
-    #                 take.markers = {}
-
-    #             take_dic["take"] = model_to_dict(take, fields=[
-    #                 "location", "duration", "rating",
-    #                 "date_modified", "markers", "id",
-    #                 "is_publish"
-    #             ])
-
-    #             takes_list.append(take_dic)
-
-    #         chunk_dic["takes"] = takes_list
-
-    #         chunk_dic2 = model_to_dict(chunk, fields=[
-    #             "startv", "endv", "id"
-    #         ])
-
-    #         chunk_dic = dict(chunk_dic.items() + chunk_dic2.items())
-
-    #         data_dic["chunks"].append(chunk_dic)
-    #     return data_dic
+        unique_together = (("version", "anthology", "language", "mode", "book"),)
+
+    # def __str__(self):

```

Comment:
why is this commented out?

Reply:
when I started to play around with the code it was giving me some error after removing the code that was commented below it, so i commented that section and it worked.  But now I believe it was just some indentation error.